src/overlays/actors/ovl_En_Dnt_Demo/z_en_dnt_demo.c

```diff
@@ -237,17 +242,22 @@ void EnDntDemo_Judge(EnDntDemo* this, PlayState* play) {
                                 break;
                         }
                         PRINTF("\n\n");
-                        // "Each index 1"
-                        PRINTF(VT_FGCOL(GREEN) "☆☆☆☆☆ 各インデックス１ ☆☆☆☆☆ %d\n" VT_RST, rand9);
-                        // "Each index 2"
-                        PRINTF(VT_FGCOL(GREEN) "☆☆☆☆☆ 各インデックス２ ☆☆☆☆☆ %d\n" VT_RST, maskIdx);
-                        // "Each index 3"
-                        PRINTF(VT_FGCOL(GREEN) "☆☆☆☆☆ 各インデックス３ ☆☆☆☆☆ %d\n" VT_RST, resultIdx);
+                        PRINTF(VT_FGCOL(GREEN) T("☆☆☆☆☆ 各インデックス１ ☆☆☆☆☆ %d\n", "☆☆☆☆☆ Each index 1 ☆☆☆☆☆ %d\n")
+                                   VT_RST,
+                               rand9);
+                        PRINTF(VT_FGCOL(GREEN) T("☆☆☆☆☆ 各インデックス２ ☆☆☆☆☆ %d\n", "☆☆☆☆☆ Each index 2 ☆☆☆☆☆ %d\n")
+                                   VT_RST,
+                               maskIdx);
+                        PRINTF(VT_FGCOL(GREEN) T("☆☆☆☆☆ 各インデックス３ ☆☆☆☆☆ %d\n", "☆☆☆☆☆ Each index 3 ☆☆☆☆☆ %d\n")
+                                   VT_RST,
+                               resultIdx);
                         PRINTF("\n");
-                        // "What kind of evaluation?"
-                        PRINTF(VT_FGCOL(YELLOW) "☆☆☆☆☆ どういう評価？  ☆☆☆☆☆☆ %d\n" VT_RST, reaction);
-                        // "What kind of action?"
-                        PRINTF(VT_FGCOL(MAGENTA) "☆☆☆☆☆ どういうアクション？  ☆☆☆ %d\n" VT_RST, this->action);
+                        PRINTF(VT_FGCOL(YELLOW) T("☆☆☆☆☆ どういう評価？  ☆☆☆☆☆☆ %d\n",
+                                                  "☆☆☆☆☆ What kind of evaluation? ☆☆☆☆☆☆ %d\n") VT_RST,
+                               reaction);
+                        PRINTF(VT_FGCOL(MAGENTA) T("☆☆☆☆☆ どういうアクション？  ☆☆☆ %d\n",
+                                                   "☆☆☆☆☆ What kind of action?          ☆☆☆ %d\n") VT_RST,
```

Comment:
strictly speaking there are two different kinds of stars here LOL
I dont really care about this but wanted to bring it to your attention. 
Also think theres a ton of spaces after "What kind of action" that arent in the original

Reply:
I don't see what you're refering to about the star types lol

The spaces are to align the output like OG did

Reply:
![image](https://github.com/user-attachments/assets/3237150a-4047-4f33-9f45-38950d753d37)